.github/stale.yml

```diff
@@ -7,7 +7,7 @@ exemptLabels:
   - pinned
   - security
 # Label to use when marking an issue as stale
-staleLabel: wontfix
+staleLabel: stale
```

Comment:
I think `wontfix` sends a dismissive message "won't ever be fixed". I like `stale + closed` --> nobody's working on this. 

`wontfix` means "rejected/out of scope" in my mind

Reply:
Agreed.  `Won't fix` means (to me) that this issue has been deemed not-compatible/not an issue while stale just means it doesn't have enough support (yet).   I like this.